src/databases/WData/wdata.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+  <Plugin name="WData" type="database" label="WDATA format reader" version="0.2.0" enabled="true" mdspecificcode="false" engspecificcode="false" onlyengine="false" noengine="false" dbtype="MTSD" haswriter="false" hasoptions="false" filePatternsStrict="false" opensWholeDirectory="false">
+    <CXXFLAGS>
+      -I./
```

Comment:
Adding the current directory as an include isn't required. It is handled automatically when the CMakeLists.txt is generated by 'xml2cmake'.

Reply:
Thats easy enough to fix

Reply:
Since I can not comment on this [response](https://github.com/visit-dav/visit/pull/17033#pullrequestreview-758766619) I will comment here.
We decided to edit _Visit_ namings to **WData**, but the library functions (see all functions in wdata.h) will still be formatted with lowercase. It is due that we have a set of tools that uses this library and changing it would produce a lot of errors. From now on all classes, methods, and others that are used in or by Visit, we name **WData**, otherwise all **wdata** lib functions are with lowercase.

Reply:
That is perfectly reasonable, thanks for clarifying. It had not been my intention to suggest changing the library function names.